lib/Tokenizer.js

```diff
@@ -635,6 +640,21 @@ Tokenizer.prototype.write = function(chunk){
 Tokenizer.prototype._parse = function(){
 	while(this._index < this._buffer.length && this._running){
 		var c = this._buffer.charAt(this._index);
+
+		/*
+		*	special conditions
+		*/
+		if(c === '`'){
+			// Toggle Markdown code to be parsed as text
+			this._isMarkdownCode = !this._isMarkdownCode;
+		} else if(c === '<' && this._index + 1 < this._buffer.length){
+			var nextChar = this._buffer.charAt(this._index + 1);
+			if(nextChar === '='){
+				// Set '<=' inequality to be parsed as text
+				this._isInequality = true;
+			}
+		}
+
```

Comment:
Hmm, this still looks out-of-place.

Let's introduce a new state `MARKDOWN` instead of tracking `this._isMarkdownCode` and `this._isInequality`.

Near top of file:

```js
MARKDOWN                  = i++,
TEXT                      = i++, // No change
```

In this function:

```js
if(this._state === MARKDOWN) {
	this._stateMarkdown(c);
} else if (this.state === TEXT) { // No change
	this._stateText(c);       // No change
```

Other functions:

```js
Tokenizer.prototype._stateMarkdown = function(c){
	if(c === '`'){
		this._state = TEXT;
	}
}

Tokenizer.prototype._stateText = function(c){
	if(c === '`'){
		this._state = MARKDOWN;
	} else if(c === "<"){
		let isInequality = (this._index + 1 < this._buffer.length) && this._buffer.charAt(this._index + 1) === '=';
		if(!isInequality){
			if(this._index > this._sectionStart){
				this._cbs.ontext(this._getSection());
			}
			this._state = BEFORE_TAG_NAME;
			this._sectionStart = this._index;
		}
	}
}
```